### PR TITLE
don't handle code link ourselves but let error list handle it

### PR DESCRIPTION
**Pull request by heejaechang**
removed our own textblock for code column in error list. rather let error list make them using information provided.

### CODE REVIEW

**Comment by heejaechang** [comment]
@srivatsn @shyamnamboodiripad @mavasani @tmeschter @jmarolf can you take a look?

**Comment by srivatsn** [comment]
:+1:

**Comment by tmeschter** [comment]
:+1:

File: src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioBaseDiagnosticListTable.cs [hunk 1]
```diff
@@ -482,60 +479,6 @@ private string GetHelpLink(DiagnosticData item)
...
-                        var item = GetItem(index);
-                        if (item == null)
-                        {
-                            return false;
-                        }
-
-                        Uri unused;
-                        if (BrowserHelper.TryGetUri(item.HelpLink, out unused))
-                        {
-                            content = GetOrCreateTextBlock(ref _errorCodes, this.Count, index, item, i => GetHyperLinkTextBlock(i, new Uri(i.HelpLink, UriKind.Absolute), bingLink: false));
```
Comment (shyamnamboodiripad) [inline src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioBaseDiagnosticListTable.cs]: Does this mean we no longer construct the bing hyperlink based on 'MessageFormat'? How does error list construct the hyperlink?

We still construct hyperlink based on MessageFormat in preview pane - do we need to change that?

Comment (heejaechang) [inline src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioBaseDiagnosticListTable.cs]: No we still construct link and give it to editor

Sent from my Windows Phone
________________________________

**Pull request merged**